nuget/helpers/build

```diff
@@ -38,3 +38,8 @@ dotnet clean
 
 echo "verifying NuGetUpdater tool"
 "$install_dir/NuGetUpdater/NuGetUpdater.Cli" --version
+
+if [ ! -f "$HOME/.nuget/NuGet/NuGet.Config" ]; then
+  echo "user-level NuGet.Config not found; credential patching will not work"
```

Comment:
It should always be there, it's really just to ensure the container was built correctly.  If the file isn't at that location, then the credential patching won't work and it'll be really difficult to figure out why.

This change is very similar to one nearer the top of the file that ensures the nuget.sln file is present; really it's just a proxy to make sure the git submodule was properly initialized, because the errors you see if not are too difficult to figure out.